apps/foundation-app/src/app/design/molecules/image-gallery/image-gallery.component.scss

```diff
@@ -0,0 +1,60 @@
+@import '~scss/styles';
+
+// parent element must have a margin-bottom of 100vw;
```

Comment:
I did some thinking about this comment I added to see if I could change it to "parent element must have a margin-bottom of its own width", but, as image-gallery stands now, I don't think it's possible. So basically, image-gallery can only be used when the active image is supposed to be the width of the screen. If image-gallery only needs to be used for galleries of images on mobile, I think this is ok. But I can think of ways to make the image-gallery adhere to the width of its parent, though it would require javascript and some document.getElementByClassNames and at least one passed in variable from the parent.